playbooks/roles/cluster-resource-override-operator/README.md

```diff
@@ -0,0 +1,75 @@
+Cluster Resource Override Operator
+================
+The **Cluster Resource Override Operator** is an admission webhook that allows you to control the level of overcommit and manage container density across all the nodes in your cluster. The Operator controls how nodes in specific projects can exceed defined memory and CPU limits.
+
+This role will deploy the Cluster Resource Override Operator by setting up the custom catalog-source with the given operator image and the ClusterResourceOverride CR with the resources overcommitment limits from the user input OR the defaults. The playbook will also run the e2e tests from the `http://github.com/openshift/cluster-resource-override-operator` repository.
+
+
+
+Requirements
+------------
+
+- Access to the cluster as a user with the cluster-admin role.
+- The cluster is in a known good state, without any errors.
+- OCP secret with name ***podman-secret*** in the default namespace which is used for global secret update and has following keys: ***username***, ***password*** and ***registry***
+
+
+Role Variables
+--------------
+
+| Variable                          | Required | Default                                                                              | Comments                                                                                                    |
+|-----------------------------------|----------|--------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------|
+| cro_role_enable                   | No       | false                                                                                | Role enablement flag to run the playbook.                                                                   |
+| cro_namespace                     | No       | clusterresourceoverride-operator                                                     | The name to be set to the namespace for operator deployment, default will be taken if not set.              |
+| cro_operator_image                | No       | `quay.io/openshift-release-dev/ocp-release-nightly:iib-int-index-art-operators-4.13` | Registry URL for operator image, the default is given Openshift version `4.13` and set for other versions . |
```

Comment:
Is there a released version image for 4.13 which we can use?

Reply:
We also use the ART image for released OCP versions, with the latest change user just need to change the OCP version instead of changing the entire image URL.  I have also updated the `README.md ` for the changed role vars.